https://lists.samba.org/archive/samba-technical/2017-December/124344.html
https://git.samba.org/?p=samba.git;a=commit;h=70f92025ae504cda97ff3a130f19035450967836

From 70f92025ae504cda97ff3a130f19035450967836 Mon Sep 17 00:00:00 2001
From: Zentaro Kavanagh <zentaro@chromium.org>
Date: Mon, 11 Dec 2017 12:41:16 -0800
Subject: [PATCH] Remove unsupported colon from configure msg.

- When cross-compiling an answers file must be supplied via
  --cross-compile --cross-answers=<path to answers>.
- The lines in the answer file have the form;
    Config Msg: Answer
- The colon is used to delimit the msg and the answer when reading
  the answers file.
- WAF doesn't support the message containing a colon.
- It's not possible to override this variable so cross compile fails.

Signed-off-by: Zentaro Kavanagh <zentaro@google.com>
Reviewed-by: Uri Simchoni <uri@samba.org>
Reviewed-by: Andrew Bartlett <abartlet@samba.org>
---
 wscript_configure_system_mitkrb5 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/wscript_configure_system_mitkrb5 b/wscript_configure_system_mitkrb5
index 9c351131ad1a..803dad7ab63b 100644
--- a/wscript_configure_system_mitkrb5
+++ b/wscript_configure_system_mitkrb5
@@ -282,7 +282,7 @@ conf.CHECK_CODE('''
        ''',
     'HAVE_WRFILE_KEYTAB',
     headers='krb5.h', lib='krb5', execute=True,
-    msg="Checking whether the WRFILE:-keytab is supported");
+    msg="Checking whether the WRFILE -keytab is supported");
 # Check for KRB5_DEPRECATED handling
 conf.CHECK_CODE('''#define KRB5_DEPRECATED 1
        #include <krb5.h>''',
-- 
2.15.1

